src/main/java/se/kth/sda6/nysure/user/UserRepository.java

```diff
@@ -1,9 +1,17 @@
 package se.kth.sda6.nysure.user;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, String> {
     User findByEmail(String email);
+
+    @Modifying
+    @Transactional
```

Comment:
The following lines prevent the app from starting altogether, therefore can't be merged as of now

Reply:
Changes made, now those lines have been deleted as they are not needed.